pkg/plugins/balance/balance.go

```diff
@@ -21,21 +22,29 @@ const (
 var _ plugins.Prioritizer = &Balance{}
 
 type Balance struct {
+	weight int32
 	handle plugins.Handle
 }
 
-func New(handle plugins.Handle) *Balance {
-	return &Balance{handle: handle}
+func New(handle plugins.Handle, weight int32) *Balance {
+	return &Balance{
+		weight: weight,
+		handle: handle,
+	}
 }
 
 func (b *Balance) Name() string {
-	return "balance"
+	return reflect.TypeOf(*b).Name()
 }
 
 func (b *Balance) Description() string {
 	return description
 }
 
+func (b *Balance) Weight() int32 {
```

Comment:
based on the discussion with @elgnay I think we do not want plugin to  provide this interface. The weight should purely controlled by  scheduler.

Reply:
ok, will remove it.